apps/chat/src/store/models/models.epics.ts

```diff
@@ -236,14 +236,23 @@ const getInstalledModelIdsFailEpic: AppEpic = (action$, state$) =>
       const recentModelIds = ModelsSelectors.selectRecentModelsIds(
         state$.value,
       );
+      const availableModels = ModelsSelectors.selectModels(state$.value);
+      const firstAvailableModelId = availableModels?.[0].id;
```

Comment:
availableModels is empty array by default, [0] can be undefined
```suggestion
      const firstAvailableModelId = availableModels[0]?.id;
```

Reply:
Please. Take reference instead of id

Reply:
Suggestion applied.
Thank you!